packages/algob/src/types.ts

```diff
@@ -443,7 +443,8 @@ export interface Deployer {
 	/**
 	 * Creates and deploys ASA defined in asa.yaml.
 	 * @name  ASA name - deployer will search for the ASA in the /assets/asa.yaml file
-	 * @flags  deployment flags */
+	 * @flags  deployment flags. Required.
+	 *   `flags.creator` must be defined - it's an account which will sign the constructed transaction. */
```

Comment:
I think we should also add a note for rekeyed accounts. In that case `ExecParams` should be used, or the direct api's should be updated to handle txns via rekeyed accounts

Reply:
hmm, maybe I'm missing something. Creator is a account with secret key. So it will have a proper secret key, right?

Reply:
replied on discord